esp/clients/ws_dfsclient/ws_dfsclient.cpp

```diff
@@ -593,11 +594,26 @@ static void configureClientSSL(IEspClientRpcSettings &rpc, const char *secretNam
     clientPrivateKeyFilename.append(tempDirStr).append("tls.key");
     caCertFilename.append(tempDirStr).append("ca.crt");
 
+    bool cacheEntryValid = false;
     CriticalBlock b(localSecretCrit);
```

Comment:
minor: This could be just before the if to reduce the window, but unlikely to make a big difference.